partiql-eval/src/main/kotlin/org/partiql/eval/PartiQLEngine.kt

```diff
@@ -31,4 +34,9 @@ public interface PartiQLEngine {
         @JvmStatic
         fun default() = PartiQLEngineBuilder().build()
     }
+
+    public class Session @OptIn(PartiQLFunctionExperimental::class) constructor(
+        var bindings: MutableMap<String, ConnectorBindings> = mutableMapOf(),
+        val functions: MutableMap<String, List<PartiQLFunction>> = mutableMapOf()
+    )
```

Comment:
At some point we'll need session variables passed during statement execution.

Reply:
Added a to-do note. Maybe we can do this after the work on Sql plugin is completed.